/robot/force_dimension:
  ros__parameters:
    sample_interval_s: 0.0005
    feedback_sample_decimation:
      pose: 50
      twist: 50
    disable_hardware: false
